flake.nix

```diff
@@ -79,7 +113,9 @@
           };
         });
 
-      checks = { } // mkCheck { system = "x86_64-linux"; username = "jente"; };
+      checks = { }
+        // mkCheck { system = "aarch64-darwin"; username = "jente"; }
+        // mkCheck { system = "x86_64-linux"; username = "jente"; };
```

Comment:
This check is currently failing due to there not being a `jente@x86_64-linux` entry for `darwinConfigurations`.